benchmark/opperf/utils/op_registry_utils.py

```diff
@@ -55,6 +57,11 @@ def _select_ops(operator_names, filters=("_contrib", "_"), merge_op_forward_back
     mx_operators = {}
     operators_with_backward = []
 
+    # Filter out deprecated operators
```

Comment:
Is there a "Deprecated" field in the operator metadata?
Because, listing operator by names is not scalable. Everytime we deprecate a operator, we need to come here to add it which soon because un-maintained.

Reply:
Did some digging around to find there's no exposed API from backend to retrieve deprecated ops. So at the moment, for the sake of this PR can we go ahead with that. I've created another issue to track the same. Sounds good?

Reply:
@sandeep-krishnamurthy